pandas/io/parsers.py

```diff
@@ -227,14 +227,19 @@
     Note that the entire file is read into a single DataFrame regardless,
     use the `chunksize` or `iterator` parameter to return the data in chunks.
     (Only valid with C parser)
+buffer_lines : int, default None
```

Comment:
well it doesn't do anything now. I guess deprecation is fine. Why don't you re-word to say it currently has no-effect.